Userland/Libraries/LibGfx/Bitmap.cpp

```diff
@@ -107,18 +107,18 @@ ErrorOr<NonnullRefPtr<Bitmap>> Bitmap::create_wrapper(BitmapFormat format, IntSi
     return adopt_ref(*new Bitmap(format, size, scale_factor, pitch, data));
```

Comment:
I guess that the commit message should be:
` LibGfx: Allow loading a Bitmap with an ideal size `

Reply:
Yep :sweat_smile: